### DISCUSSION TITLE
How to monitor if DBA Dash is in complete stopped state or not collecting data for any of the onboarded servers

### DISCUSSION THREAD

**Discussion by IndrajitMurgude** [trimble-oss/dba-dash#1130] [General]
is there any query which I can run on DBADash database and then write a custom code on top of it to get an email alert ?

Also, on 6th of Nov, DBA Dash stopped working, I do not have the logs now, but I feel it may be because of day light time change.

**Comment by DavidWiseman** [comment]
You can query the dbo.CollectionDates table which has a list of each instance, collection type, and the date it was last collected.  There is also a dbo.CollectionDatesStatus view that adds the threshold information.  Dates are stored in UTC format.

I haven't seen the DBA Dash service crash before - it could happen, but it should log any errors and keep running.  If it happens again, check the logs folder and also check the eventlog which might contain details about an unhandled exception that crashed the process.  You can also check the service recovery options in services.msc.  It can be configured to restart the service in the case of failure.

Hope this helps
